test_tools/process_client.py

```diff
@@ -786,3 +786,13 @@ def get_workspace_publication_thumbnail(publication_type, workspace, name, *, ac
     response = requests.get(r_url, headers=headers, timeout=HTTP_TIMEOUT)
     raise_layman_error(response)
     return response.content
+
+
+def workspace_exists(workspace):
+    with app.app_context():
+        return workspace in layman_util.get_workspaces(use_cache=False)
+
+
+def user_exists(username):
```

Comment:
In process_client, we usually have methods for calling Layman REST API endpoints and retrieving data, not methods for testing if some publication/user/workspace exist. Therefore, I prefer to move methods `workspace_exists` and `user_exists` to file delete_user_test.py.

It would be also nice to create method `get_users` in process_client.py that will call GET Users endpoint and return the result. This `get_users` method can be later used in `user_exists` instead of `layman_util.get_usernames`.